README_SFRAME_FORMAT.rst

```diff
@@ -1,15 +1,14 @@
-Blosc2 Extended-Frame Format
-============================
+Blosc2 Sparse Frame Format
+==========================
 
-Blosc (as of version 2.0.0) has an extended-frame format that allows for the storage of different Blosc data chunks
-sparse, on-disk.
+Blosc (as of version 2.0.0) has a sparse frame (sframe for short) format that allows for the storage of different Blosc data chunks on disk in a sparse way.
 
-When creating an eframe one must denote the `storage.sequential` as false and provide a name (which will be a directory)
-in `storage.urlpath` for the eframe to be stored. It is recommended to name the directory with the `.b2eframe`
+When creating an sparse frame one must denote the `storage.sequential` as false and provide a name (which will be a directory)
+in `storage.urlpath` for the sframe to be stored. It is recommended to name the directory with the `.b2sframe`
 extension.
 
-An eframe is made up of a frame file and the chunks stored in the same directory on-disk.
-The frame follows the format described in the `frame format <README_FRAME_FORMAT.rst>`_ document, with the difference
+An sframe is made up of a frame file and the chunks stored in the same directory on-disk.
```

Comment:
Consider:

... is made up of _multiple files: a frame index chunk and individually stored data chunks in the same directory on-disk_. 

In this way:
frame index chunk (or index chunk for short as also referred to it further in the document)
data chunks = as distinguished from frame index chunk.

Reply:
done.